sdk/translation/azure-ai-translation-text/samples/sample_text_translation_break_sentence.py

```diff
@@ -0,0 +1,81 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_text_translation_break_sentence.py
+
+DESCRIPTION:
+    This file contains sample snippets for the Text Translation service.
+
+USAGE:
+    python sample_text_translation_break_sentence.py
+
+    Set the text translation endpoint environment variables with your own value before running the samples:
+        
+        1) AZURE_TEXT_TRANSLATION_ENDPOINT - the endpoint to your Text Translation resource.
+        Note: the endpoint must be formatted to use the custom domain name for your resource:
+        https:\\<NAME-OF-YOUR-RESOURCE>.cognitiveservices.azure.com\
+                    
+    The create_text_translation_client_with_credential call requires additional variables:
+        2) AZURE_TEXT_TRANSLATION_APIKEY - the API key to your Text Translation resource.
+        3) AZURE_TEXT_TRANSLATION_REGION - the Azure Region of your Text Translation resource.
+"""
+
+from azure.core.exceptions import HttpResponseError
+from azure.ai.translation.text.models import (InputTextItem)
+
+# -------------------------------------------------------------------------
+# Text translation client
+# -------------------------------------------------------------------------
+import sample_text_translation_client
+text_translator = sample_text_translation_client.create_text_translation_client_with_credential()
```

Comment:
It's a better experience for customers to have the client created in each sample, then they dont have to go look for this specific client creation sample.

Reply:
same feedback for all of the samples

Reply:
You can take inspiration from the Document Translation library

Reply:
I have created a task for this change.  The C# snippets don't include the client creation code, so we should change all of the samples so they are consistent.

https://machinetranslation.visualstudio.com/MachineTranslation/_workitems/edit/148604/